doc/release-notes.md

```diff
@@ -156,7 +221,29 @@ RPC Deprecations
 - `z_gettotalbalance` has been deprecated. Use `z_getbalanceforaccount` instead.
 - `dumpwallet` has been deprecated. Use `z_exportwallet` instead.
 
+Build System
+------------
+
+- Clang has been updated to use LLVM 13.0.1.
+- libcxx has been updated to use LLVM 13.0.1, except on Windows where it uses 13.0.0-3.
+- The Rust toolchain dependency has been updated to version 1.59.0.
+
 Platform Support
 ----------------
 
 - Debian 9 has been removed from the list of supported platforms.
+- A build issue (a missing header file) has been fixed for macOS targets.
+- On Arch Linux only, use Debian's libtinfo5_6.0 to fix a build regression.
+
+Mining
+------
+
+- Mining to Orchard recipients is now supported on testnet. 
```

Comment:
recipients -> receivers? Or "Mining to a Unified Address that contains an Orchard receiver" possibly.

Reply:
The reason I used "recipients" here is that, although the only way to construct an Orchard recipient address is a unified address containing an Orchard receiver, that's an implementation detail.